packages/clay-checkbox/src/index.tsx

```diff
@@ -60,13 +60,10 @@ const ClayCheckbox: React.FunctionComponent<Props> = ({
 	return (
 		<div
 			{...containerProps}
-			className={classNames(
-				'custom-control custom-checkbox',
-				containerProps.className,
-				{
-					'custom-control-inline': inline,
-				}
-			)}
+			className={classNames('custom-control custom-checkbox', {
+				[`${containerProps.className}`]: containerProps.className,
```

Comment:
Instead of putting this in an object, I think you can just do 

```js
classNames(
     'custom-control custom-checkbox',
     containerProps.className,
     {
          'custom-control-inline': inline
     }
);
```